docs/content/v0.20/packaging.md

```diff
@@ -144,12 +136,17 @@ The dashed dependencies are at run time, not build time.
 
 "KCM" is the KubeStellar controller-manager.
 
-"ks_scripts" are the user-facing instructions that use published
```

Comment:
Not part of this PR, but I don't think that anyone is using this acronym (KCM)   for the few places we refer to it we can go with "KS controller maanger"

Reply:
Don't forget to think about the verbosity of the labels in the diagrams.